packages/core/src/utils/graphql.ts

```diff
@@ -136,25 +171,45 @@ export function findDeprecatedUsages(schema: GraphQLSchema, ast: DocumentNode):
           const parentType = typeInfo.getParentType();
           if (parentType) {
             const reason = fieldDef.deprecationReason;
+            const extensions: DeprecationExtensions = {
+              type: 'fieldDeprecation',
+              parentType: parentType.name,
+              fieldDef: fieldDef.name,
+            };
             errors.push(
               new GraphQLError(
                 `The field '${parentType.name}.${fieldDef.name}' is deprecated.${reason ? ' ' + reason : ''}`,
-                [node]
+                [node],
+                undefined,
+                undefined,
+                undefined,
+                undefined,
+                extensions
               )
             );
           }
         }
       },
       EnumValue(node) {
-        const enumVal = typeInfo.getEnumValue();
```

Comment:
This part hasn't changed. Just named the variables a little better for consistency with the above sections